src/nus/cs2103/addressbook/AddressBook.java

```diff
@@ -699,16 +791,6 @@ private static void clearAddressBook() {
         savePersonsToFile(getAllPersonsInAddressBook(), storageFilePath);
     }
 
-    /**
-     * Resets the internal model with the given data. Does not save to file.
-     *
-     * @param persons list of persons to initialise the model with
-     */
-    private static void initialiseAddressBookModel(List<String[]> persons) {
-        ALL_PERSONS.clear();
-        ALL_PERSONS.addAll(persons);
-    }
-
```

Comment:
This means other places can call `ALL_PERSONS` `List` methods? I worry that students will see it and add a new CRUD feature which changes the master list directly, and forget to trigger a save to file.